docs/source/operations/conversions/unitconvert.rst

```diff
@@ -124,12 +128,30 @@ The same list can also be produced on the command line with :program:`proj` or
 | ind-ch   | Indian Chain                    |
 +----------+---------------------------------+
 
+.. _angular_units:
+
+Angular units
+###############################################################################
+
+In the table below all angular units supported by PROJ `unitconvert` are listed.
+(since PROJ 5.2.0)
```

Comment:
Please use `.. versionadded:: 5.2.0` instead. Place it below the section header. That's consistent with the rest of the docs.

Reply:
Done